docs/source/set_dev_env.rst

```diff
@@ -41,19 +41,26 @@ Create database and user::
 For convenience for development, user, password and database name are already
 set in `tcms/settings/devel.py` with default value. Each of them is `nirrate`.
 
-Load database schema::
+.. note::
 
-    mysql -uroot nitrate < contrib/sql/nitrate_db_setup.sql
+    You may want to adjust the database and/or other configuration settings.
+    Override them in ``./tcms/settings/devel.py`` if necessary. While the
+    Nitrate team prefers MySQL, sqlite has been verified to work for development
+    and some people have reported using PostgreSQL in production!
```

Comment:
I have one instance of Nitrate with PostgreSQL and it is running fine regardless of the hard-coded SQL statements. I agree that the SQl makes it less portable though. Maybe a sentence highlighting this fact needs to be added, what do you think ?